job_executor/config/log.py

```diff
@@ -0,0 +1,115 @@
+import json
+import logging
+import logging.handlers
+import multiprocessing
+import platform
+import sys
+import traceback
+from datetime import datetime
+from multiprocessing import Queue
+
+import tomlkit
+from json_logging import util
+
+
+class ContextFilter(logging.Filter):
+    """
+    This is a filter which injects contextual information into the log.
+    """
+    process_name = ""
+
+    def __init__(self, process_name):
+        super().__init__()
+        self.process_name = process_name
+
+    def filter(self, record):
+        # prefix record.msg instead of adding a new field
+        # to be compliant with Kibana
+        record.msg = self.process_name + ': ' + record.msg
+        return True
+
+
+def configure_worker_logger(queue: Queue):
```

Comment:
Is there a way we could get the jobId for the running job in the log here too? 🤔
(Does not need to be part of this PR, but maybe a task for the future)

Reply:
Sure, I can add it as a prefix to message instead of process name as I don't think we need both.